.gitignore

```diff
@@ -1,3 +1,4 @@
+/.angular/cache
```

Comment:
```suggestion
.angular/
```
should be probably just `.angular`
https://github.com/angular/components/blob/master/.gitignore#L14

Reply:
Thanks, will check. That's what cli13 is generating